src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,123 @@ The accordion component lets users show and hide sections of related content on
 
-You should also take into account the number of sections of content – accordions display vertically, so they can fit more sections than horizontal tabs.
+## Decide between using accordions, tabs and details
 
-Tabs may work better for users who need to switch quickly between 2 sections. Accordions push other sections down the page when they open, but tabs do not move which makes it easier to switch.
+Accordions, [tabs](/components/tabs/) and [details](/components/details/) all work by hiding sections of content which a user can choose to reveal. Be careful when using any of these components within one another.
 
-Consider using an accordion instead of the [details](/components/details/) component if there are multiple related sections of content. The details component might be better if you only have one or two sections of content. The details component is less visually prominent than an accordion, so tends to work better for content which is not as important to users.
+If you decide to use one of these components, consider if:
+- the user needs to look at more than one section at a time — an accordion can show multiple sections at a time, unlike tabs
+- the user needs to switch quickly between sections — tabs can show content without pushing other sections down the page, unlike accordions
+- there are many sections of content — accordions can fit more sections as they’re arranged vertically, unlike tabs which are arranged horizontally
+- there’s only one or two pieces of short, less important content — the details component is more suitable as it’s visually smaller and less prominent than an accordion or tabs
 
 ## How it works
 
 There are 2 ways to use the accordion component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
-### Find out how your users interact with the content
+### Section heading buttons
+
+An accordion will usually start with all sections hidden. To show a section, the user can interact anywhere in the heading button.
+
+The heading button includes all of these areas:
+- heading text
+- summary text (if you decide to add one)
+- call-to-action text to 'show' or 'hide'
+
+For users of screen readers, all the text in the button will be read as a single statement (separated by commas to allow for slight pauses). There’s also some visually hidden content in the heading text to help announce the call-to-action as 'show this section' or 'hide this section'
+
+{{ example({group: "components", item: "accordion", example: "default", html: true, nunjucks: true, open: false, size: "xl"}) }}
+
+#### Write clear button text
+
+Write the heading and summary text like any other button text and:
+- use sentence case
+- describe the content that will show
+- keep it short
 
```

Comment:
I think that's a good call @EoinShaughnessy! Given the option between making them sentences and the gerunds, I'd go with sentences. The bullet lists are getting a bit repetitive too.

```suggestion
Write the heading and summary text like any other button text. Use sentence case, describe the content that will show, and keep it short."
```